test/package.js

```diff
@@ -21,3 +21,21 @@ test('Every rule is defined in index file', async t => {
 	t.is(Object.keys(index.configs.recommended.rules).length, ruleFiles.length,
 		'There are more exported rules in the recommended config than rule files.');
 });
+
+test('Every fixable rule got valid meta.type', async t => {
+	const validTypes = ['problem', 'suggestion', 'layout'];
+
+	const files = await pify(fs.readdir)('rules');
+	const ruleFiles = files.filter(file => path.extname(file) === '.js');
+
+	for (const file of ruleFiles) {
+		const name = path.basename(file, '.js');
+		const rule = index.rules[name];
+
+		t.true(rule.meta !== null, `${name} got no meta`);
+		if (rule.meta.fixable) {
```

Comment:
I think we should require `meta.type` for all rules, even if they're not fixable. They might become fixable in the future and we might also use the data for other things, like categorization.